frontend/src/reducers/staticContentReducer.js

```diff
@@ -37,7 +38,7 @@ const staticContentReducer = (state = initialState, action) => {
     case actionTypes.STORE_TENURE_TYPES:
       return {
         ...state,
-        mineTenureTypes: action.payload.options,
+        mineTenureTypes: createDropDownList(action.payload, "description", "mine_tenure_type_code"),
```

Comment:
whats the purpose of this change?

Reply:
This was outlined in developers by Jason yesterday in an effort to decouple the front end from the back end.

Reply:
Follow up: I don't see any changes to the selectors, the `getMineTenureTypes` selector was used to create several others, including a dropdown.

Reply:
Im not sure I understand what you mean?

Reply:
This is the opposite of what we decided in the developers channel. 
data transformations should occur in the selectors, not when they are being saved in the store.

Reply:
The data returned`getMineTenureTypes` was used to create 4 other selectors, including dropdown, a hash object, conditional rendering for commodity/disturbances.

Reply:
I agree. However, this was the smallest change that would allow for the swagger refactor to work while also de-coupling the backend from the front end in this instance. The rest of the selector issues can be batch fixed as this isn't the only instance of this.

Reply:
If I understand this all correctly, I see Justin's point here. `action.payload.options` used to return an array of objects with `label`s and `value`s, correct? And so, in order to maintain backwards compatibility, `createDropDownList` is added here to format the new API response into the "old" format that the frontend uses.

With that said, Meredith is also correct. This is the opposite of how we want to do our data transformation. We want to store it as-is and then format it on extraction from the store. This should be as simple as updating the selector to do what you've done here instead of formatting it in the reducer.

Also, Justin, I've left a comment about the response that you're providing from this endpoint. Please take a look at that feedback alongside this comment.

Does that all make sense to everyone?

Reply:
This is an effort to bring the backend up to standard, but the FE expected the value in the store to be the correct form for the dropdown, this is the smallest change to fix the backend without breaking the frontend. I think someone needs to take a pass these outstanding issues to bring them in line. But I don't think we want Justin going down all these rabbit holes in this PR/Ticket.

Reply:
ah, yes. Thanks, @NWCalvank for chiming in.
My point is specifically about the location of the transformation.
Happy with the decoupling of the BE & FE.

Reply:
@Jsyro I'm all for bringing up the backend standard, but not at the expense of the frontend standards we have for the reducers/selectors. 

Please store the data as is, and make the transformation in the correct location.

Reply:
Not the hill I would die on given the scope of changes Justin has been working on, and it can be updated in another branch as this code doesn't get chiseled in stone :) That said, this should be a one liner in the same file. Justin, if you don't think making this change will cause a lot of regression testing for you I would make it. Otherwise, we can note it with a comment or in our backlog.

Reply:
I thought this was a breaking change until Nathans comment. I see now that it doesn't break anything, so disregard my previous comments. 

I'm very pleased with the work Justin is doing, but I'd also to avoid adding unnecessary tickets into the backlog, in order to decouple the BE and FE there will need to be changed made to both and I'd like to see the standards being met for both. 

That is all, not blocking this PR, happy coding.